source/blog/2017-02-01-testing-rdo-with-tempest-new-features-in-ocata.html.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Testing RDO with Tempest: new features in Ocata'
+author: ltoscano
+tags: tempest,install,configure,rpm,ocata
+date: 2017-02-01 14:15:19 CET
+---
+
+The release of Ocata, with its shorter release cycle, is close and
+it is time to start a broader testing (even if one could argue that
+it is always time for testing!).
+
+One of the core pieces for testing the cloud is the
+[Tempest](http://docs.openstack.org/developer/tempest/overview.html).
+
+# Tempest in RDO
+
+## Current status
+
+The status up to the Newton is well described in few blog posts,
+either from [packages](../../2016/09/running-tempest-on-rdo-openstack-newton/)
+or from [git](../../2016/11/how-to-install-and-run-tempest/).
+In short, RDO used a [forked repository](https://github.com/redhat-openstack/tempest),
+which regularly received all the changes from the official Tempest.
+The main reason for this was a configuration script which auto-discovers
+the features of the cloud (with some hints based on the version) and creates
+a valid `tempest.conf`.
+
+## Changes in Ocata
+
+The auto-configuration script was decoupled from the internal Tempest and
+moved to the new [python-tempestconf](https://github.com/redhat-openstack/python-tempestconf) repository,
+thanks to the work of Martin Kopec, Chandan Kumar and Daniel Mellado.
+
+This means less burden work required (no need to keep the fork) but also
+simplifies a bit the steps required to initialize and run Tempest tests,
+close to the process documented by Tempest upstream.
+
+## New configuration steps
+
+Configure the RDO repositories, then install the required packages:
+
+    $ yum -y install python-tempestconf openstack-tempest
```

Comment:
$ it should be $sudo yum -y install openstack-tempest, python-tempestconf is a dependency of openstack-tempest so no need to install python-tempestconf.